db/migrate/20190627150411_remove_trusted_applications.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveTrustedApplications < ActiveRecord::Migration[5.2]
+  def change
+    drop_table :trusted_applications
+  end
+end
```

Comment:
these are irreversible, do we care about that?

Reply:
I'm thinking it was never-used-feature, but I'm afraid to connect to the production DB to verfy that. :-)

Reply:
The only thing that used them was... the view that allowed you to create/destroy them.
So yeah they were unused.
All registered apps are currently allowed to read all exercises (and solutions)
If we want to limit this in the future, delegating read permissions to apps would be more powerful than this TrustedApplication thing.